conf/cassandra.yaml

```diff
@@ -1752,6 +1752,17 @@ drop_compact_storage_enabled: false
 # write_consistency_levels_warned: []
 # write_consistency_levels_disallowed: []
 #
+# Guardrail to warn or fail when writing column values larger than threshold.
+# This guardrail is only applied to the values of regular columns because both the serialized partitions keys and the
+# values of the components of the clustering key already have a fixed, relatively small size limit of 65535 bytes, which
+# is probably lesser than the thresholds defined here.
+# This guardrail is different to max_value_size. max_value_size is checked when deserializing any value to detect
+# sstable corruption, whereas this guardrail is checked on the CQL layer at write time to reject regular user queries
+# inserting too large columns.
+# Default -1 to disable.
```

Comment:
`Invalid data storage: value must be non-negative` will be thrown if you do that, as you use `DataStorageSpec.LongBytesBound` which doesn't allow `null`

Reply:
I have review in progress, just haven’t submitted it yet. This is just wording, he uses null internally :-) needs to be corrected only in the text

Reply:
Right, that comment should come from an older version of the patch. Changed to `The two thresholds default to null to disable`.